tools/background_removal/background_removal.xml

```diff
@@ -0,0 +1,55 @@
+<tool id="background_removal" name="Background Removal" version="0.1">
+    <description>with scikit-image</description>
+    <requirements>
+        <requirement type="package" version="0.24.0">scikit-image</requirement>
+    </requirements>
+    <command detect_errors="aggressive">
+        <![CDATA[
+        python "$__tool_directory__/background_removal.py" $input_image $filter $radius $output
+        ]]>
+    </command>
+    <inputs>
+        <param name="input_image" type="data" format="tiff, jpg, jpeg, png, tif" label="Input Image"/>
+        <param name="filter" type="select" label="Select Filter">
+            <option value="rolling_ball">Rolling-Ball Algorithm</option>
+            <option value="dog">Difference of Gaussians</option>
+            <option value="top_hat">Top-Hat Filter</option>
+        </param>
+        <param name="radius" type="float" label="Radius" value="20"/>
+    </inputs>
+    <outputs>
+        <data name="output" format="tiff" label="Background Substraction Output"/>
```

Comment:
```suggestion
        <data name="output" format="tiff" label="Background substraction output"/>
```
This is more consistent with the other tools.

Reply:
Done!